image-layout.md

```diff
@@ -156,4 +161,44 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## index.json
+
+This OPTIONAL file is for use indexing references and elements of the image-layout, where walking the `refs` is sub-optimal.
+It is intended to be generated from the content in an image-layout after any changes.
+
+The [manifest-list](manifest-list.md) is an multi-descriptor entry point.
+This index provides an established path (`/index.json`) to have an entry point for this image-layout and discover auxiliary descriptors pertaining to the references.
+
+### Index Example
+
+```json,title=Manifest%20List&mediatype=application/vnd.oci.image.manifest.list.v1%2Bjson
+{
+  "schemaVersion": 2,
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "size": 7143,
+      "digest": "sha256:e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7fc331f",
+      "annotations": {
+        "org.opencontainers.ref.name": "v1.0"
```

Comment:
> how would you refer to a given revision of an image w/o a ref name?

* By index in the array (e.g. `manifests[0]`), or
* By another annotation (e.g. `com.example.foo.bar="docs"`), or
* By media type (e.g. `application/vnd.example.docs+xml`).

Consumers which were based on referencing solely via `org.opencontainers.ref.name` would just ignore (and be unable to address) entries which didn't set that annotation.